compass/ocean/tests/global_ocean/files_for_e3sm/remap_sea_surface_salinity_restoring.py

```diff
@@ -29,11 +29,11 @@ def __init__(self, test_case):
                          ntasks=512, min_tasks=1)
 
         self.add_input_file(
-            filename='PHC2_salx.2004_08_03.filled_double_precision.nc',
-            target='PHC2_salx.2004_08_03.filled_double_precision.nc',
+            filename='woa23_decav_0.25_sss_monthly_extrap.20241031.nc',
+            target='woa23_decav_0.25_sss_monthly_extrap.20241031.nc',
```

Comment:
The actual file in the initial condition database needs to be renamed `woa` --> `woa23`.

Reply:
I can't do it because I don't have write permission. In general, if you can chmod all your files in `mpas_standalone` so they are group writable, that would be helpful.

Reply:
Interesting!  I can rename it without having write permission.

Reply:
Sorry for the stupid question, but here you mean just make sure group has full rwx permissions after I put the file in the initial_condition_database and not figuring how to do this through the python script?

Reply:
No, these files should not be executable.  Just group readable and writable.  (Folks add execute permission to files a lot that shouldn't have it and this is not a great idea.)  To change permissions recursively, do something like:
```
chmod -R g+rwX .
```
The capital `X` indicates execute permission only gets added to directories, not files.

Reply:
> not figuring how to do this through the python script?

Correct, not in the python script.

Reply:
I caught my 'x' and did not add it when I changed permissions.  It should just be 'rw' for groups.